egs/aspire/s5/local/nnet3/run_ivector_common.sh

```diff
@@ -3,14 +3,16 @@
 # this script is based on local/multicondition/run_nnet2_common.sh
 # minor corrections were made to dir names for nnet3
 
-stage=0
+stage=1
+snrs="20:10:15:5:0"
 foreground_snrs="20:10:15:5:0"
 background_snrs="20:10:15:5:0"
 num_data_reps=3
 db_string="'air' 'rwcp' 'rvb2014'" # RIR dbs to be used in the experiment
                                       # only dbs used for ASpIRE submission system have been used here
 RIR_home=db/RIR_databases/ # parent directory of the RIR databases files
 download_rirs=true # download the RIR databases from the urls or assume they are present in the RIR_home directory
+base_rirs="simulated"
```

Comment:
directly provide the rir_set_parameters here for generating the new results and the add in comments the rir_set_parameters for generating the old results.